cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -875,7 +850,7 @@ pub mod bridging {
 				1,
 				[
 					Parachain(SiblingBridgeHubParaId::get()),
-					PalletInstance(parachains_common::rococo::snowbridge::INBOUND_QUEUE_PALLET_INDEX)
+					PalletInstance(testnet_parachains_constants::rococo::snowbridge::INBOUND_QUEUE_PALLET_INDEX)
```

Comment:
Cleaner if imported on top with `use testnet_parachains_constants::rococo::snowbridge`

Reply:
ok, thank you, fixed here: https://github.com/paritytech/polkadot-sdk/pull/3134/commits/406c5bf199a90838bb90dcd23d7bea89eb434ce9